### PR TITLE
AppImage: fix variable syntax

### PR DESCRIPTION
**Pull request by git-developer**

### CODE REVIEW

**Comment by v1993** [comment]
This shouldn't be needed? The code in question is adapted directly from [example in docs](https://appimage-builder.readthedocs.io/en/latest/reference/recipe.html#section-scripts) and uses a variable set by appimage builder itself.

Oh well, let's check what artifacts will be.

**Comment by v1993** [comment]
Aand it works. Sigh, can people at least test code before putting it into documentation?..

File: dist/AppImageBuilder.yml [hunk 1]
```diff
@@ -18,8 +18,7 @@ AppDir:
     exec_args: $@
 
   after_runtime: |
-    echo "X-AppImage-Integrate=false" | tee -a "${TARGET_APPDIR}/org.v1993.evdevhook2.desktop"
-    echo "Test file" | tee -a "${TARGET_APPDIR}/test_file.txt"
+    echo "X-AppImage-Integrate=false" | tee -a "{{APPIMAGE_PATH}}/org.v1993.evdevhook2.desktop"
```
Comment (v1993) [inline dist/AppImageBuilder.yml:21]:
```suggestion
    echo "X-AppImage-Integrate=false" >> "{{APPIMAGE_PATH}}/org.v1993.evdevhook2.desktop"
```

Should work just as well with less log spam.

Comment (git-developer) [reply dist/AppImageBuilder.yml]: Of course. I just fixed the command.

**Comment by git-developer** [comment]
> Or might be because of appimage-builder action using an outdated version of program.

Yes, looks like `build-appimage@v1` is not "latest v1" but "1.0". It was [never updated](https://github.com/AppImageCrafters/build-appimage/issues/4). `build-appimage@v1.3` uses `appimage-builder:1.1.0`, so maybe an update avoids issues like this one.

**Pull request merged**